README.md

```diff
@@ -140,47 +149,59 @@ If you have any issues with Reco or you just want the latest news on the develop
 * **[Wlan Signal](https://github.com/Arvinth-Krishna/Reco-PC-Server#-Wlan-Signal--)**
 * **[WhatsApp](https://github.com/Arvinth-Krishna/Reco-PC-Server#-whatsapp--)**
 * **[YouTube (search)](https://github.com/Arvinth-Krishna/Reco-PC-Server#-youtube-search--)**
-
-
+</blockquote></details>
+	
 
 ## Features Documentation:  [↟](https://github.com/Arvinth-Krishna/Reco-PC-Server#quick-jumps) 
-
+<details><summary>Click to expand!</summary><blockquote>
+		
 ### ★ Abort  [⇪](https://github.com/Arvinth-Krishna/Reco-PC-Server#features-list--)
```

Comment:
you have not closed this segment with /details and /blockquote

Reply:
Thanks for that. I was skimming through it to get it done.